app/v1/services/SubmitReturnService.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.services
+
+import cats.data.EitherT
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.Logging
+import v1.connectors.SubmitReturnConnector
+import v1.controllers.EndpointLogContext
+import v1.models.errors._
+import v1.models.outcomes.ResponseWrapper
+import v1.models.request.SubmitReturnRequest
+import v1.models.response.SubmitReturnResponse
+import v1.support.DesResponseMappingSupport
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SubmitReturnService @Inject()(connector: SubmitReturnConnector) extends DesResponseMappingSupport with Logging {
+
+  def submitReturn(request: SubmitReturnRequest)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext,
+    logContext: EndpointLogContext): Future[Either[ErrorWrapper, ResponseWrapper[SubmitReturnResponse]]] = {
```

Comment:
```suggestion
    logContext: EndpointLogContext): Future[Either[ErrorWrapper, ResponseWrapper[Unit]]] = {
```

See connector comments